docs/reference/mapping/types.asciidoc

```diff
@@ -75,7 +75,7 @@ markup. Used for identifying named entities.
 <<completion-suggester,`completion`>>:: Used for auto-complete suggestions.
 <<search-as-you-type,`search_as_you_type`>>:: `text`-like type for
 as-you-type completion.
-<<semantic-text, `semantic_text`>>:: 
+<<semantic-text, `semantic_text`>>:: Used for performing semantic search.
```

Comment:
Do we have an overview we can link to here that also says what semantic search is?

Reply:
Added via [6a086bc](https://github.com/elastic/elasticsearch/pull/108311/commits/6a086bc02815060ae56a3320aea9e0ab23d413c9).